resolwe_bio/processes/samtools/samtools_mpileup.py

```diff
@@ -0,0 +1,346 @@
+"""Samtools mpileup."""
+
+import re
+from pathlib import Path
+
+import pandas as pd
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    FileField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+def parse_mpileup_entry(row):
```

Comment:
@jkokosar I can add additional tool parameters as input fields, that change the behaviour of the ``base_sequence`` column. For example, excluding the deletions in the output.